flink-runtime/src/main/java/org/apache/flink/runtime/io/disk/FileChannelManagerImpl.java

```diff
@@ -51,12 +54,27 @@
 	/** The number of the next path to use. */
 	private volatile int nextPath;
 
+	/** Prefix of the temporary directories to create */
+	private final String prefix;
+
+	/** Flag to signify that the file channel manager has been shut down already. */
+	private final AtomicBoolean isShutdown = new AtomicBoolean();
```

Comment:
I think we should give some comments for the reason of atomic here, which is for the case of multiple shutdown hooks.

Reply:
I have added the comments.

Reply:
typo signify